src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs

```diff
@@ -89,6 +89,7 @@ protected override async Task CleanupClientConnections(string fromInstanceId = n
                     connectionIds = _connectionIds.Where(s => s.Value == fromInstanceId).Select(s => s.Key);
                 }
 
+                Log.StartToCleanupClientConnection(Logger, _connectionIds.Count);
```

Comment:
only log when count > 0?

Reply:
if count ==0 , the code will return earlier.
https://github.com/Azure/azure-signalr/blob/5c945df8ba0f357e6a7244ab75084c67aa058318/src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs#L82-L92